spec/factories/items.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
-  factory :item, class: 'Cocina::Models::RequestDRO' do
+  factory :persisted_item, class: 'Cocina::Models::RequestDRO' do
```

Comment:
For naming consistency / clarity, can we use `dro` instead of `item`?

Reply:
Oh, I take that back. This does following naming conventions https://github.com/sul-dlss/cocina-models#usage-conventions